templates/zerver/help/deactivate-or-reactivate-a-user.md

```diff
@@ -28,6 +28,33 @@ user's bots will also be deactivated. Lastly, the user will be unable to
 create a new Zulip account in your organization using their deactivated
 email address.
 
+### Notify users of their deactivation
+
+If you want to notify the user of their deactivation,
+you can do so by sending them a email.
```

Comment:
Zulip can optionally send the user an email notification that their account was deactivated.